src/Contract/Operation/Duplicateable.php

```diff
@@ -32,10 +32,8 @@ interface Duplicateable
      *
      * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#duplicate
      *
-     * @template U
-     *
-     * @param null|callable(U): (Closure(U): bool) $comparatorCallback
-     * @param null|callable(T, TKey): U $accessorCallback
+     * @param (callable(mixed): (Closure(mixed): bool)) $comparatorCallback
+     * @param callable(T, TKey): mixed $accessorCallback
```

Comment:
these seem better done in a separate PR as they don't seem related to the iterators work.

Also, again, needs to be in line with the operation itself; but in the operation the template `U` is still there: https://github.com/loophp/collection/pull/233/files#diff-91532943eb0b11a568f0217fd47cb706a322494a906f5f2884ee76a9d71e60d1R30

Reply:
Ok !